lldb/source/Breakpoint/BreakpointLocation.cpp

```diff
@@ -508,8 +508,20 @@ void BreakpointLocation::GetDescription(Stream *s,
         s->PutCString("re-exported target = ");
       else
         s->PutCString("where = ");
+
+      // If there's a preferred line entry for printing, use that.
+      bool show_function_info = true;
+      if (GetPreferredLineEntry()) {
+        sc.line_entry = *GetPreferredLineEntry();
```

Comment:
nit: avoid 2 function calls
```suggestion
      if (auto may_be_line_entry = GetPreferredLineEntry()) {
        sc.line_entry = *may_be_line_entry;
```